src/main/kotlin/org/radarbase/redcap/util/RedCapTrigger.kt

```diff
@@ -160,11 +161,12 @@ class RedCapTrigger(value: String) {
                         instrumentStatus(
                             value.substring(markerIndex + 1).toInt()
                         )
+                else -> {}
```

Comment:
isn't this implied?

Reply:
i would handle the specific case, `null`, instead of using open else. This makes the code easier to understand.

Reply:
Ah yes there were cases where the `convertParameter()` returned null. Updated it to handle null specifically.